mmadu-identity/src/main/resources/templates/login.html

```diff
@@ -6,7 +6,9 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <title>Mmadu Login</title>
+    <title th:if="${domainConfiguration.theme.loginTitle == null}">Mmadu Login</title>
```

Comment:
@IEnoobong I think you should create a domainConfiguration.displayName instead, its more general and we can use that in other places. The theme object is just for the appearance of the page.

Reply:
Got it!